Adding bundle cli command documentation (#4881)
open-policy-agent/opa: docs/content/management-bundles.md

See the [Configuration Reference](../configuration) for configuration details.

### Bundle build

> The CLI command [`opa build`](https://www.openpolicyagent.org/docs/latest/cli/#opa-build) gives you the capability to build your own bundles.

Reviewer: Let's use relative links instead: `[...](../cli/#opa-build)` please -- also below.

> Here is a basic example on how to build a bundle from a folder called `foo`. The bundle will be named by default `bundle.tar.zg`.

Reviewer: ```suggestion
Here is a basic example on how to build a bundle from a folder called `foo`. The bundle will be named by default `bundle.tar.gz`.
```

> ```

Reviewer: Let's add `console` here for markup of console interactions.

```
```console
```

$ ls
example.rego

$ opa build -b foo/
```

More, you can optimize the bundle by specifying the `--optimize` or `-O` flag.
```
opa build -b foo/ --optimize=1
```

> Finally, you can also sign your bundle thanks to `opa build`.

Reviewer: ```suggestion
Finally, you can also sign your bundle with `opa build`:
```
